src/widget_window.c

```diff
@@ -103,51 +103,53 @@ GtkWidget *widget_window_create(
 
 	GtkLayerShellLayer layer = GTK_LAYER_SHELL_LAYER_ENTRY_NUMBER;
 
-	value = get_tag_attribute(attr, "layer");
-	if (value) {
```

Comment:
```
if (!attr)
    goto layer_set;
```

Reply:
I'll try it, but probably should have been a forethought.